scheduler/src/test/java/org/apache/falcon/notification/service/SchedulerServiceTest.java

```diff
@@ -305,6 +307,25 @@ public PRIORITY getPriority() {
         ((MockDAGEngine)mockDagEngine).removeFailInstance(instance1);
     }
 
+    @Test
+    public void testResume() throws Exception {
+        storeEntity(EntityType.PROCESS, "summarize6");
+        Process mockProcess = getStore().get(EntityType.PROCESS, "summarize6");
+        mockProcess.setParallel(1);
+        Date startTime = EntityUtil.getStartTime(mockProcess, cluster);
+        ExecutionInstance instance1 = new ProcessExecutionInstance(mockProcess, new DateTime(startTime), cluster);
+        Properties resumeProps = new Properties();
+        resumeProps.setProperty(FalconWorkflowEngine.FALCON_RESUME, "true");
+        instance1.setProperties(resumeProps);
+        instance1.setExternalID("123");
+        SchedulerService.JobScheduleRequestBuilder request = (SchedulerService.JobScheduleRequestBuilder)
+                scheduler.createRequestBuilder(handler, instance1.getId());
+        request.setInstance(instance1);
+        scheduler.register(request.build());
+        Thread.sleep(100);
+        Assert.assertEquals(((MockDAGEngine)mockDagEngine).getTotalResumes(instance1), new Integer(1));
```

Comment:
This is the unit test for SchedulerService which is not aware of statuses of instances. For the status check I have updated InstanceSchedulerManagerJerseyIT accordingly.